src/editors/_shared/hooks/DataTypes/types/ArrayType.js

```diff
@@ -3,37 +3,12 @@ import DataType from "../AbstractDataType";
 import { checkIfDefaultOrDisabled } from "./utils";
 
 class ArrayType extends DataType {
-  // Array type properties definition
   key = DATA_TYPES.ARRAY;
   label = "Array";
-  default = "[]";
+  default = [];
 
-  editComponent = (props, mode = "row") => {
-    const editor = {
-      row: _props => this.stringEditComponent(_props, this.default),
-      dialog: this.codeEditComponent
-    };
-    return editor[mode](props);
-  };
-
-  /**
-   * Validate array value
-   * @param {*} value
-   * @returns
-   */
-  validate(value) {
-    value = value?.replace(/'/g, '"');
-    return new Promise(resolve => {
-      try {
-        if (checkIfDefaultOrDisabled(value)) {
-          return resolve({ success: true, value });
-        }
```

Comment:
The bug that I mention in my final comment, might be avoided with this check here.

Reply:
I don't know what you mean